invenio_app_ils/acquisition/pidstore/pids.py

```diff
@@ -16,3 +16,12 @@
 
 VENDOR_PID_FETCHER = "venid"
 """Fetcher PID for Vendor."""
+
+ACQUISTION_PURCHASE_ORDER_PID_TYPE = "apordid"
```

Comment:
nit: I would go for `acqoid`, and for the vendors above `acqvid`, just to "namespace" them